### DISCUSSION TITLE
no available server to connect.

### DISCUSSION THREAD

**Discussion by Gunvior** [apache/incubator-seata#5877] [Q&A] [answered]
如何解决：
已经启动nacos和seata。seata版本0.9 
![image](https://github.com/seata/seata/assets/87658514/d62a40a1-f7f8-434d-9277-7c0418689885)
![image](https://github.com/seata/seata/assets/87658514/9c5e3c72-4850-4d94-a742-ff899657b92a)
![image](https://github.com/seata/seata/assets/87658514/5a56ff52-96e8-444a-8c9b-3e61a8a3e211)
![image](https://github.com/seata/seata/assets/87658514/6f4699d6-9222-4ad6-9fbd-97c324ecb313)

报错:  ERROR |timeoutChecker_1 |NettyClientChannelManager.java:166 |io.seata.core.rpc.netty.NettyClientChannelManager |no available server to connect.

**Comment by slievrly** [comment, marked as answer]
https://seata.io/zh-cn/docs/user/txgroup/transaction-group
